.github/workflows/linux_riscv_xuantie.yml

```
@@ -0,0 +1,256 @@
+name: Linux RISC-V with Xuantie (Ubuntu 22.04, Python 3.10)
```

I'd suggest to add version of rvv to the job name because we have 2 cmake toolchains files for different vector instruction sets : for rvv0.7.1 and rvv1.0. In the current dockerfile we cross-compile OpenVINO for RVV0.7.1.

```suggestion
name: Linux RISC-V RVV 0.7.1 with Xuantie (Ubuntu 22.04, Python 3.10)
```

---

We would like to reduce the workflow names, because they are cropped in PR jobs list. 
btw, how users could switch between rvv0.7.1 and rvv1.0?

---

There are 2 cmake toolchain files: for [rvv071](https://github.com/openvinotoolkit/openvino/blob/master/cmake/toolchains/riscv64-071-xuantie-gnu.toolchain.cmake) and for [rvv10](https://github.com/openvinotoolkit/openvino/blob/master/cmake/toolchains/riscv64-100-xuantie-gnu.toolchain.cmake). The user should know which rvv version is supported by their device

---

Then it is better to put the version into the build job name